cves/CVE-2016-6817.yml

```diff
@@ -204,4 +206,5 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: This is a very good example of why coding with "magic numbers" can
+    lead to issues in the future
```

Comment:
Agreed. I don't see the section about the lessons from class. Was it not a part of this YAML file?

Reply:
The "lessons from class" segment was already in this YAML file, as well as a few others, but it seems like only some of them have this section